src/parser.rs

```diff
@@ -210,6 +228,19 @@ impl_rdp! {
             (_: include_tag, &name: string) => {
                 Ok(Some(Node::Include(name.replace("\"", "").to_string())))
             },
+            (_: import_macro_tag, &tpl_name: string, &name: simple_ident) => {
+                Ok(Some(Node::ImportMacro {
+                    tpl_name: tpl_name.replace("\"", "").to_string(),
```

Comment:
`tpl_name.trim_matches("\"").to_string()` conveys more information, I think. It's also faster.